R/pkg/R/sparkR.R

```diff
@@ -93,7 +93,7 @@ sparkR.stop <- function() {
 #' sc <- sparkR.init("local[2]", "SparkR", "/home/spark",
 #'                  list(spark.executor.memory="1g"))
 #' sc <- sparkR.init("yarn-client", "SparkR", "/home/spark",
-#'                  list(spark.executor.memory="1g"),
+#'                  list(spark.executor.memory="4g", spark.driver.memory="2g"),
```

Comment:
I could take this out - this is more for API doc/roxygen. If we are moving it out it would be visible only when someone is reading the code. Maybe pointing to the SparkR programming guide is better.